pkgs/by-name/vg/vgmtrans/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  autoPatchelfHook,
+  qt6,
+}:
+stdenv.mkDerivation {
+  pname = "vgmtrans";
+  version = "1.1-unstable-2024-07-31";
+
+  src = fetchFromGitHub {
+    owner = "vgmtrans";
+    repo = "vgmtrans";
+    rev = "9de4c06b03f8c9ec9ecb5c81dfcb53d1b8f75bbe";
+    hash = "sha256-689ceDbQrifH64eA79fCpa+eB70gJYKSyiZDlC1HZiA=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    autoPatchelfHook
```

Comment:
Why do we need this if we build from source?

Reply:
Because of the libbass vgmtrans shipped with. The one in Nixpkgs has been marked broken for a very long time and the project expects it to be dynamically linked.